.gitignore

```diff
@@ -41,3 +41,5 @@
 ehthumbs.db
 Thumbs.db
 
+dbInfo.json
```

Comment:
why is this here?

Reply:
I added it to my git ignore because it somehow got removed